src/InvoiceGenerator/InvoiceGenerator.BusinessLogic/ClientService.cs

```diff
@@ -0,0 +1,49 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace InvoiceGenerator.BusinessLogic
+{
+    public class ClientService
+    {
+        public List<ClientViewModel> GetClients()
+        {
+            using (var context = new InvoiceDBContext()) // using statement is discarded once the squiggly brackets have been passed, saves memory
```

Comment:
Just a comment to say that we'll look at using dependency injection to inject the InvoiceDbContext directly into the Business Logic layer soon.